### ISSUE TITLE
how do you add search to organic

### ISSUE THREAD

**Issue by markmark1** [mamaral/Organic#12]
how do you add searchbar to organic uitableview
thanks for an amazing library

**Comment by mamaral** [comment]
You could simply add it as a header view for your table.

**Comment by markmark1** [comment]
That's not the problem how about the filtered search data source how about loading from backend 

Can you please give examples since it's not in read me can't use it till given

**Comment by markmark1** [comment]
Pls give an example with organic

**Comment by mamaral** [comment]
Are you familiar with how to do this with a normal `UITableView`?

**Comment by markmark1** [comment]
Yes but not sure how that would translate into organic ; can you please share  the right way to do with organic Also it's not quite clear how the data source is handled in cells with reuse 

On Tue, Jun 28, 2016 at 7:10 PM -0700, "Mike Amaral" notifications@github.com wrote:

Are you familiar with how to do this with a normal UITableView?

**Comment by markmark1** [comment]
Another thing that would be good in an example is load more or append more cells by loading data from backend how would you do that with organic as the table view interface is not exposed directly 

On Tue, Jun 28, 2016 at 6:29 PM -0700, "Mike Amaral" notifications@github.com wrote:

You could simply add it as a header view for your table.

**Comment by markmark1** [comment]
Amazing library can't wait to use for everything also what do you think of component kit

**Comment by mamaral** [comment]
Take a look at the demo project - that uses sections using reuse which basically exposes the cellForRowAtIndexPath as a block.

**Comment by markmark1** [comment]
I did but it doesn't show how todo load more and search 

On Tue, Jun 28, 2016 at 7:29 PM -0700, "Mike Amaral" notifications@github.com wrote:

Take a look at the demo project - that uses sections using reuse which basically exposes the cellForRowAtIndexPath as a block.

**Comment by markmark1** [comment]
@mamaral  would it be possible to give a gist of how to add search code snippets to organic using 

--- Take a look at the demo project - that uses sections using reuse which basically exposes the cellForRowAtIndexPath as a block.
